INTITA/Modules/Profile/Cells/ProfileHeaderView.swift

```diff
@@ -8,10 +8,13 @@
 import UIKit
 
 protocol ProfileHeaderViewDelegate: AnyObject {
-    func editImage()
     func avatarTapped()
 }
 
+protocol ProfileHeaderViewDataSourse: AnyObject {
+    func editImage()
+}
+
 class ProfileHeaderView: UITableViewCell {
```

Comment:
Please, name class with "...Cell" at the end

Reply:
done